python/Ganga/new_tests/TestProxy.py

```diff
@@ -79,7 +85,7 @@ def test_dict_attributes(self):
         self.assertFalse(hasattr(proxied, '_impl'))
 
         # A proxy instance should only have _impl in its dict
-        self.assertEqual(self.p.__dict__.keys(), ['_impl'])
+        self.assertEqual(self.p.__dict__.keys(), ['_impl', '_proxyObject'])
```

Comment:
Wait, there is something concerning here.
If line #73 fails and is assertTrue it created a proxyObject attribute for the instance which effects this test.
(which when 73 was different I had to modify this)

I'm slightly bugged that this will pass/fail or change what is a pass or fail based on line #73. It should stand by itself as a pass/fail.